docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
-          have to delete the ``wallet`` folder and its identities prior to re-running the javascript application
-          or you will get an error. This happens because the Certificate Authority and its database are taken down
-          when the test-network is taken down but the original wallet still remains in the application-javascript directory
-          so it must be deleted.  When you re-run the sample javascript application, a new wallet and credentials will
-          be generated.
+.. code-block:: bash
 
+  npm start
 
-If you scroll back up to the beginning of the output in your terminal, it should be similar to below:
 
-.. code:: bash
-
-  Wallet path: /Users/<your_username>/fabric-samples/asset-transfer-basic/application-javascript/wallet
-  Successfully enrolled admin user and imported it into the wallet
-
-Because the admin registration step is bootstrapped when the Certificate Authority
-is started, we only need to enroll the admin.
-
-.. note:: Since the Fabric CA interactions are common across the samples, enrollAdmin() and the other CA
-          related functions are included in the ``fabric-samples/test-application/javascript/CAUtil.js``
-          common utility.
-
-As for the app user, we need the application to register and enroll the user in the next step.
-
-
-Second, the application registers and enrolls an application user
------------------------------------------------------------------
+First, the application creates a gRPC connection to the Gateway peer
+--------------------------------------------------------------------
 
-Now that we have the administrator's credentials in a wallet, the application uses the ``admin``
-user to register and enroll an app user which will be used
-to interact with the blockchain network. The section of the application code is shown below.
+The client application establishes a `gRPC <https://grpc.io/>`__ connection to the Gateway peer that it will use to
+transact with the network. To do this it only requires the endpoint address of the Gateway peer and, if the Gateway is
+configured to use TLS, appropriate TLS certificates to establish a secure connection. There is significant overhead
+associated with establishing gRPC connections, so this connection should be retained by the application and shared by
+all Gateway connections to the Gateway peer.
+
+.. note:: In order to maintain security of any private data used in transactions, the application should connect to a
+          Gateway peer belonging to the same organization as the client identity. If the client identity's organization
+          does not host any peers then a Gateway peer hosted by an upstream organization through which the client's
+          organization is authorized to transact should be used.
+
+The TypeScript application creates a gRPC connection using the TLS certificate of the Certificate Authority for the
```

Comment:
The TypeScript application creates a gRPC connection using the TLS certificate from the target peer organization's Certificate Authority to verify the authenticity of the peer's credentials.
(@bestbeforetoday this may need work but you get the idea to try to condense it.)